### ISSUE TITLE
StreamCorruptedException exception on startup

### ISSUE THREAD

**Issue by JerryShea** [OpenHFT/Chronicle-Queue#558]
```
We got an exception on startup of our system today (running Chronicle BOM 1.16.153) :

...
[2018-12-25 17:26:54.073077] Caused by: java.lang.IllegalStateException: java.io.StreamCorruptedException: Data at 264 overwritten? Expected: 0 was 40010024
[2018-12-25 17:26:54.073086]    at net.openhft.chronicle.queue.impl.single.SingleChronicleQueueExcerpts$StoreAppender$StoreAppenderContext.close(SingleChronicleQueueExcerpts.java:782)
[2018-12-25 17:26:54.073095]    at net.openhft.chronicle.queue.impl.single.SingleChronicleQueueExcerpts$StoreAppender$StoreAppenderContext.close(SingleChronicleQueueExcerpts.java:728)
...
...
[2018-12-25 17:26:54.073154]    ... 10 more
[2018-12-25 17:26:54.073164] Caused by: java.io.StreamCorruptedException: Data at 264 overwritten? Expected: 0 was 40010024
[2018-12-25 17:26:54.073174]    at net.openhft.chronicle.wire.AbstractWire.updateHeaderAssertions(AbstractWire.java:530)
[2018-12-25 17:26:54.073183]    at net.openhft.chronicle.wire.AbstractWire.updateHeader(AbstractWire.java:516)
[2018-12-25 17:26:54.073192]    at net.openhft.chronicle.queue.impl.single.SingleChronicleQueueExcerpts$StoreAppender$StoreAppenderContext.close(SingleChronicleQueueExcerpts.java:757)
[2018-12-25 17:26:54.073201]    ... 16 more
```

**Comment by JerryShea** [comment]
Possibly resolved by #561, #580, #559, #551 ?

**Comment by JerryShea** [comment]
Closing as this has not reoccured since delivery of a later CQ with the above improvements

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.17.11](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.17.11), [BOM-2.17.206](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.206)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.17.11](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.17.11), [BOM-2.17.225](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.225)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.17.11](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.17.11), [BOM-2.17.241](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.241)
